ui/pages/asset/components/token-buttons.tsx

```diff
@@ -95,8 +93,10 @@ const TokenButtons = ({
 
   const account = useSelector(getSelectedInternalAccount, isEqual);
 
-  const { chainId: multichainChainId } =
-    useMultichainSelector(getMultichainNetwork);
+  // This is only returning solana mainnet, not solana devnet?
```

Comment:
Dummy, I forgot to remove this comment 🙊 

I'll do a cleanup PR on this with the PR below.

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/32720